canonical-web-and-design/react-components
# Add SummaryButton component for ModularTable

## Done

- Adds `ShowMore` component to be used with modular table to load more items

<img width="1052" alt="Screenshot 2020-10-07 at 15 18 45" src="https://user-images.githubusercontent.com/83575/95336022-6b284680-08b0-11eb-988b-2ebe877f77c6.png">


## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Go to [storybook page for ShowMore component](https://react-components-300.demos.haus/?path=/docs/summarybutton--default-story)
- Check if examples look and work as expected
- Check full example with modular table: https://react-components-300.demos.haus/?path=/docs/modulartable--load-more

## Fixes

Fixes: https://github.com/canonical-web-and-design/vanilla-squad/issues/902
